src/ludamus/templates/panel/_personal_data_fields.html

```
@@ -0,0 +1,69 @@
+            {% elif field.field_type == "select" %}
+                <select name="{{ name_prefix }}_{{ field.slug }}"
+                        id="id_{{ name_prefix }}_{{ field.slug }}"
+                        class="w-full border border-border rounded-lg px-4 py-2 focus:outline-none focus:ring-2 focus:ring-primary">
+                    <option value="">—</option>
+                    {% for option in field.options %}
+                        <option value="{{ option.value }}"
+                                {% if current_value == option.value %}selected{% endif %}>{{ option.label }}</option>
+                    {% endfor %}
+                </select>
+                {% if field.allow_custom %}
+                    <input type="text"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)